src/module/StaticPage/config/module.config.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @author    Benjamin Knorr (benjamin@serlo.org]
+ * @copyright 2017 by www.serlo.org
+ * @license   LGPL
+ * @license   http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License (LGPL]
+ */
+namespace StaticPage;
+
+return [
+    'di' => [
+        'allowed_controllers' => [
+            __NAMESPACE__ . '\Controller\StaticPageController'
+        ],
+        'definition' => [
+            'class' => [
+                __NAMESPACE__ . '\Controller\StaticPageController' => []
+            ]
+        ]
+    ],
+    'router' => [
+        'routes' => [
+            'deutsch' => [
+                'type' => 'hostname',
+                'options' => [
+                    'route' => 'de.:domain.:tld',
```

Comment:
Just to confirm, this is intentional? Could you link me to the docs?

Reply:
Yes, as I understood it this matches the `de` subdomain, and ignore the params `:domain` and `:tld` since they aren't constrained (as e.g. `de.serlo.org` as route wouldn't match our local `de.serlo.localhost`, and if we one day change our domain this should still work^^)

Docs are here: https://framework.zend.com/manual/2.0/en/modules/zend.mvc.routing.html#zend-mvc-router-http-hostname

Reply:
If there is another way to check the subdomain let me know!

Reply:
I'm not sure it works like that, you did not supply any pattern matches. Does this work locally?

Reply:
It worked with `de.serlo.localhost:4567/spenden` but i couldn't test if `en.serlo.localhost:4567/spenden` doesn't

Reply:
See #617 for issue with en.serlo.localhost

Reply:
ping @arekkas